Provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.kt

```diff
@@ -49,52 +51,58 @@ class ConfidenceFeatureProvider private constructor(
         }
     }
 
+    private fun startListeningForContext() {
+        coroutineScope.launch {
+            confidence.contextChanges
+                .drop(1)
+                .distinctUntilChanged()
+                .collect {
+                    resolve(InitialisationStrategy.FetchAndActivate)
```

Comment:
What if the strategy was set to `ActivateAndFetch`? Somewhat makes me question the whole thing about fetch strategy.

Reply:
[here](https://github.com/spotify/confidence-openfeature-provider-kotlin/blob/cf8f712a141862f8d54f1c70593625b65770d754/Provider/src/main/java/com/spotify/confidence/ConfidenceFeatureProvider.kt#L71) we are resolving first with the current context and the given initialization strategy. then we start listening for the changes, if any change is happening, we refresh the cache after resolving (same logic as in main). I moved the drop and distinct to the confidence object so it's more clear that the `contextChanges` is only about changes from this point of time onwards.

Reply:
In Swift we don't force this to be `FetchAndActivate`: https://github.com/spotify/confidence-openfeature-provider-swift/blob/0d1cefdeb766a3d24c7d05be5f834d8855f271f3/Sources/ConfidenceProvider/ConfidenceFeatureProvider.swift#L107
Is it expected? 
(I am also noticing that the `strategy` argument is not used in the linked Swift function)

Reply:
that's a bug in swift, i fixed it [here](https://github.com/spotify/confidence-openfeature-provider-swift/pull/101), good catch!